m_tunnel_key.go

```diff
@@ -0,0 +1,144 @@
+package tc
+
+import (
+	"fmt"
+	"net"
+
+	"github.com/mdlayher/netlink"
+)
+
+const (
+	tcaTunnelUnspec = iota
+	tcaTunnelKeyTm
+	tcaTunnelKeyParms
+	tcaTunnelKeyEncSrc
+	tcaTunnelKeyEncDst
+	tcaTunnelKeyEncKeyID
+	tcaTunnelKeyPad
+	tcaTunnelKeyEncDstPort
+	tcaTunnelKeyNoCSUM
+	tcaTunnelKeyEncOpts
+	tcaTunnelKeyEncTOS
+	tcaTunnelKeyEncTTL
+)
+
+// TunnelKey contains attribute of the TunnelKey discipline
+type TunnelKey struct {
+	Parms           *TunnelParms
+	Tm              *Tcft
+	KeyEncSrc       *net.IP
+	KeyEncDst       *net.IP
+	KeyEncKeyID     *uint64
```

Comment:
`TCA_TUNNEL_KEY_ENC_KEY_ID` seems to be `u32`
https://elixir.bootlin.com/linux/latest/source/net/sched/act_tunnel_key.c#L338